new-machine.md

```diff
@@ -43,13 +43,17 @@
 
 9. Set up public DNS for the machine.
 
-10. Run the install:
+10. Edit `/etc/puppet/modules/compbox/manifests/init.pp` to change the `$compstate` variable to point at your compstate.
```

Comment:
This should not be needed. It is expected and intended that the default deploy is the dummy compstate. You should then just deploy your actual one over the top (hence the comment about being aware of the warnings on the first deploy).

Reply:
Ahh apologies, missed that when skimming the README, shall read in more detail next time!

Reply:
0e2dc43

Reply:
Hrm, I wonder if the last step in the machine setup should be something like "now deploy your compstate using `srcomp deploy` locally"?

Reply:
That sounds like a good idea!

Reply:
051257e